src/client.ts

```diff
@@ -54,7 +54,8 @@ export const DefaultUser: UserInfo = {
   attributes: {},
 };
 
-export interface ReactSDKClient extends Omit<optimizely.Client, 'createUserContext' | 'getProjectConfig'> {
+export interface ReactSDKClient
```

Comment:
Was this the linter moving this to the next line?  

Actually, I think it's probably ok to expose `isOdpIntegrated` to React SDK.